dev-rust/cargo_c/cargo_c_bin-0.9.27.recipe

```diff
@@ -16,10 +16,13 @@ elif [ "$targetArchitecture" = x86_64 ]; then
 	SOURCE_URI="https://github.com/Begasus/cargo-c/archive/$srcGitRev.tar.gz"
 	CHECKSUM_SHA256="0af0cef06db1aa7d46ca6ce60ad89bc1ca0c3705c51a7f80f6f87e0b2b84d8f7"
 	SOURCE_DIR="cargo-c-$srcGitRev/64bit"
+else
+	SOURCE_URI=""
```

Comment:
lint error: list index out of range
But if I put this outside of the if loop I get that error here localy?

Reply:
Might also add CHECKSUM_SHA256

Reply:
Didn't solve lint error (works locally) :/

Reply:
works now